app/controllers/api/autocompletes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::AutocompletesController < ApplicationController
+  def index
+    render json: autocomplete_response
+  end
+
+private
+
+  def autocomplete_response
+    Services
+      .search_api_v2
+      .autocomplete(params.require(:q))
```

Comment:
It feels like we should add a request test that asserts that we get a bad_request response if there isn't a q parameter.